core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -576,10 +576,15 @@ public GraphHopper init(CmdArgs args) {
             CGIARProvider cgiarProvider = new CGIARProvider();
             cgiarProvider.setAutoRemoveTemporaryFiles(args.getBool("graph.elevation.cgiar.clear", true));
             tmpProvider = cgiarProvider;
+        } else if (eleProviderStr.equalsIgnoreCase("gmted")) {
+            tmpProvider = new GMTEDProvider();
+        } else if (eleProviderStr.equalsIgnoreCase("multi")) {
+            tmpProvider = new MultiSourceElevationProvider();
         }
 
         tmpProvider.setCalcMean(eleCalcMean);
-        tmpProvider.setCacheDir(new File(cacheDirStr));
+        if (!cacheDirStr.isEmpty())
```

Comment:
Currently the default cacheDir was "" (the project root) and not "/tmp/[provider-name]". I changed that, but I am not sure if this might create issues for users of non unix systems?

Reply:
Yeah, we should keep it to project root

Reply:
It looks like we use /tmp/cgiar already so `/tmp/<provider-name>` makes sense.